Compatibility issues with ganache v7.0.1

**Bug description**

The test at `tests/test_uniswap.py::TestUniswap::test_get_fee_maker[1]` always results in an error due to refusal to establish a connection with the local target machine:

```
During handling of the above exception, another exception occurred:
E   urllib3.exceptions.NewConnectionError: <urllib3.connection.HTTPConnection object at 0x000001873E1940A0>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it
```

While trying to pinpoint the exact error, I noticed that the option `--miner.legacyInstamine true` was not available for my ganache version v7.0.1

**Reproduction**

Setup:
- Windows 10
- ganache v7.0.1

Run `make test` 

**Solution Proposal**

Replace `--miner.legacyInstamine true` with either `--miner.instamine "eager"` or `--miner.instamine "strict"` in  `tests/test_uniswap.py`
```
p = subprocess.Popen(
        f"""ganache
        --port {port}
        --wallet.seed test
        --chain.networkId 1
        --chain.chainId 1
        --fork.url {os.environ['PROVIDER']}
        --miner.defaultGasPrice {defaultGasPrice}
        --miner.instamine "eager"
        """.replace(
            "\n", " "
        ),
        shell=True,
    )
```

--- a/.github/workflows/test.yml
+++ b/.github/workflows/test.yml
@@ -48,8 +48,7 @@ jobs:
     - name: Install dependencies
       run: |
         poetry install
-        # TODO: Update to stable ganache when released!
-        npm install -g ganache@beta
+        npm install -g ganache@7.0.1
     - name: Test
       env:
         PROVIDER: ${{ secrets.MAINNET_PROVIDER }}

--- a/tests/test_uniswap.py
+++ b/tests/test_uniswap.py
@@ -94,7 +94,7 @@ def ganache() -> Generator[GanacheInstance, None, None]:
         --chain.chainId 1
         --fork.url {os.environ['PROVIDER']}
         --miner.defaultGasPrice {defaultGasPrice}
-        --miner.legacyInstamine true
+        --miner.instamine "strict"
         """.replace(
             "\n", " "
         ),

